src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/flow_units/ResourceFlowUnit.java

```diff
@@ -27,8 +28,19 @@
 import org.jooq.Field;
 import org.jooq.impl.DSL;
 
+/**
+ * ResourceFlowUnit is the flowunit type that is emitted by RCA vertex.
+ * It is persisted in the FlowUnit SQLite table
+ *
+ * <p>The SQL table name  : FlowUnit
```

Comment:
can we change the name of the table to RCA otherwise, people looking at schema wouldn't know what that table means ?

Reply:
done. changed to RCA